src/browser/types.ts

```diff
@@ -122,9 +122,11 @@ declare global {
 
             // TODO: describe executionContext more precisely
             executionContext: (RunnerTest | RunnerHook) & {
-                hermioneCtx: {
-                    assertViewResults: Array<AssertViewResult>;
-                };
+                testplaneCtx: ExecutionThreadToolCtx;
+                /**
+                 * @deprecated Use `testplaneCtx` instead
+                 */
+                hermioneCtx: ExecutionThreadToolCtx;
```

Comment:
This label would style every usage with strikethrough with specific label:
<img width="668" alt="Screenshot 2024-04-03 at 02 34 59" src="https://github.com/gemini-testing/hermione/assets/54893992/acd6049d-4f83-4c2c-866d-17149a6e168b">